src/assets/source/scripts/modules/serlo_breadcrumbs.js

```diff
@@ -0,0 +1,181 @@
+/**
+ *
+ * Athene2 - Advanced Learning Resources Manager
+ *
+ * @author  Jonas Keinholz (jonas.keinholz@serlo.org)
+ * @license LGPL-3.0
+ * @license http://opensource.org/licenses/LGPL-3.0 The GNU Lesser General Public License, version 3.0
+ * @link    https://github.com/serlo-org/athene2 for the canonical source repository
+ *
+ * Breadcrumb
+ *
+ */
+
+/*global define*/
+define('breadcrumbs', ['jquery'], function ($) {
+    'use strict';
+    var Breadcrumbs,
+        instance,
+        defaults;
+
+    defaults = {
+        // main wrapper selector
+        wrapperId: '#subject-nav-wrapper',
+
+        // breadcrumb selector
+        breadcrumbId: '#breadcrumbs',
+
+        // desired height
+        height: '45'
+    };
+
+    /**
+     * @class Breadcrumbs
+     * @param {Object} options See defaults
+     *
+     * Main constructor
+     **/
+    Breadcrumbs = function (options) {
+        if (!(this instanceof Breadcrumbs)) {
```

Comment:
I was bold and just copied the template of [serlo_side_navigation](https://github.com/serlo-org/athene2/blob/e412973a353fdcd31d02006a35dc5a0c4d0486d9/src/assets/source/scripts/modules/serlo_side_navigation.js). But I think that you don't need the line, even for the singleton pattern (this is already handled by the returned function). Do you remember any reason why it was done in serlo_side_navigation?